backend/routes/userData.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+const userDataRouter = express.Router()
+
+userDataRouter.get('/', (req, res) => {
```

Comment:
kind of a strange route imo - the full path for this is `wewrap.com/user-data/` which in traditional REST nomenclature would normally be used for a bulk GET endpoint, aka GET me many users. the correct way to structure this endpoint is as `/userData/:id`.

Reply:
done!

Reply:
From the frontend, we wouldn't have a user id to send to the backend, that's why I did `/userData/`. The route just returns the user from the cookie.